### DISCUSSION TITLE
Some misconfigured lights

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13787] [Bug reports]
### Disclaimers  

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened? 

That light on Typhon 1 airlock is just off. I think it should be on and shadow casting like on Typhon 2 for example 
![20240417223145_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/cf70818d-3087-4506-9766-cbc90561f88e)
![20240417230524_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/7e2007ff-3edc-43d4-a509-6cd2ed057487)

Those 2 red lights on dugongs back should have smaller range i think. Shadow casting/ draw behind sub looks uglier imho
![20240417223951_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/61b14416-a28d-4216-bb4a-8eac91fc8190)
![20240417223848_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/f2ca407a-db75-4170-966d-e508c1315a60)
Also turn this on, and probably copy paste from some other marker light coz it's color and range are default
![20240419031006_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/9dbd1a5a-e742-4a53-9fcf-ea001d0566ef)
Game litteraly unplayable, fix zis

### Reproduction steps

Go check

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by NotWendy** [comment]
If someone decides to assign this ticket to themselves, they may as well examine all lights on vanilla submarines. This report is a random selection of them. Plenty of other lights go through walls in random places. Emergency lights especially suffer from that.

**Reply by SomeRandomNoobKekeke** [reply]
I'm pretty sure that's all of them

Or did you mean lights that shine from in to out?

**Reply by SomeRandomNoobKekeke** [reply]
Ok, i see
Whoever dare to assign to this, you can use [this lua script](https://steamcommunity.com/sharedfiles/filedetails/?id=3226171203) to see them, or [these screenshots](https://drive.google.com/drive/folders/1b2-ETRsY4fLnf_L-ggAu57u1cIZrX27f?usp=drive_link)
I only really care about the ones mentioned above
![barodev small](https://github.com/FakeFishGames/Barotrauma/assets/122838333/09882b22-447b-4e0b-b2a8-0690bd9b7e2f)

**Comment by Regalis11** [comment]
I would not consider these bugs. It's highly subjective what the "correct" way to set up lights is: it's not a bug if a sub builder wants to go performance first and make a light non-shadow-casting even if it means it bleeds through a wall. Even more so when it comes to what kind of a light range "looks ugly" or whether some light components is supposed to be a decorative element with a very short range or something that actually illuminates it's surroundings.

**Reply by SomeRandomNoobKekeke** [reply]
Airlock light on typhon is off for performance reasons, i get it

![barodev](https://github.com/FakeFishGames/Barotrauma/assets/122838333/9790a972-6f3c-4417-a3b7-fec08b7e260e)

**Reply by SomeRandomNoobKekeke** [reply]
and giving those lights on dugong smaller range so they won't bleed inside would also ruin performance
![barodev big](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b8f0de2d-0fce-4f70-a1e5-79e88e5b3cb4)

**Reply by NotWendy** [reply]
It's important that we don't fix these issues in vanilla submarines, else people will start expecting actually decent quality from them and then we'll have to start addressing the sprite depth problems. :BaroDev

**Comment by Regalis11** [comment]
Decided to address this anyway, commit in our private repo: https://github.com/FakeFishGames/Barotrauma-development/commit/b0fdb01b0ead01eb03d91cc387b830f002a3a6ca
